actors/runtime/src/main/java/cloud/orbit/actors/cloner/JavaSerializationCloner.java

```diff
@@ -28,30 +28,33 @@
 
 package cloud.orbit.actors.cloner;
 
+import cloud.orbit.actors.runtime.BasicRuntime;
+import cloud.orbit.actors.runtime.JavaMessageSerializer;
+
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
-import java.io.ObjectInputStream;
-import java.io.ObjectOutputStream;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
 
 /**
  * Java serialization based object cloning implementation.
  */
-public class JavaSerializationCloner implements ExecutionObjectCloner
+public class JavaSerializationCloner extends JavaMessageSerializer implements ExecutionObjectCloner
```

Comment:
allows for compatibility with cloud.orbit.actors.runtime.JavaMessageSerializer.ReferenceReplacement.